src/components/Comment/Comment.scss

```
@@ -0,0 +1,9 @@
+.post__comment {
+  margin: 20px 10px;
+  padding: 20px 10px;
+
+  background-color: #ebb294;
+  border: 1px solid rgb(185, 89, 155);
+  border-radius: 20px;
+  box-shadow: 0 0 10px rgba(0,0,0,0.5);
+}
```

Додай пустий рядок в кінець файлу
```suggestion
}

```